src/main/java/org/jenkinsci/maven/plugins/hpi/TestDependencyMojo.java

```diff
@@ -244,19 +273,25 @@ public void execute() throws MojoExecutionException {
             }
 
             /*
-             * At this point, the model has been updated as the user has requested. We now redo
-             * resolution and compare the new resolution to the original in order to account for
-             * updates to transitive dependencies that are not present in the model. Anything that
-             * was updated in the new resolution needs to be updated in the test classpath. Anything
-             * that was removed in the new resolution needs to be removed from the test classpath.
+             * At this point, the model has been updated as the user has requested. We now
+             * redo
+             * resolution and compare the new resolution to the original in order to account
+             * for
+             * updates to transitive dependencies that are not present in the model.
+             * Anything that
+             * was updated in the new resolution needs to be updated in the test classpath.
+             * Anything
+             * that was removed in the new resolution needs to be removed from the test
+             * classpath.
```

Comment:
I find the changes in this file make it less readable rather than more readable.  Wrapping paragraphs at different line lengths does not seem like a significant help for readability.  I think you should revert the changes to this file.  If that level of white space change is needed, I would prefer it be implemented with the maven `spotless` plugin as is done for a number of other Jenkins repositories.
```suggestion
             * At this point, the model has been updated as the user has requested. We now redo
             * resolution and compare the new resolution to the original in order to account for
             * updates to transitive dependencies that are not present in the model. Anything that
             * was updated in the new resolution needs to be updated in the test classpath. Anything
             * that was removed in the new resolution needs to be removed from the test classpath.```